src/main/java/org/broadinstitute/hellbender/utils/test/BaseTest.java

```diff
@@ -53,6 +53,10 @@ public void setTestVerbosity(){
     public static final String publicTestDir = new File(gatkDirectory, publicTestDirRelative).getAbsolutePath() + "/";
     public static final String publicTestDirRoot = publicTestDir.replace(publicTestDirRelative, "");
 
+    public static final String moduleTestDataDir = publicTestDir + "org/broadinstitute/hellbender/";
+    public static final String toolTestDataDir = moduleTestDataDir + "tools/";
+    public static final String bqsrTestDataDir = toolTestDataDir + "BQSR/";
```

Comment:
We shouldn't put tool-specific constants up here in `BaseTest`. Things like `toolTestDataDir` are fine, but things like `bqsrTestDataDir` should live down in the tool-specific test classes